### PR TITLE
Monitoring documentation

### PR DESCRIPTION
**Pull request by antimony123**
- Revised monitoring and grafana docs, aggregated them into one document
- Fixed tiny change in monitoring_piman.sh to reflect recent changes in monitoring-client.py

Please note that this documentation does not cover any automation scripts which have been merged after this pull request was made. Once they have been merged, this documentation will be updated to reflect the new changes.

### CODE REVIEW

**Comment by NickFulton98** [comment]
Nice work!

**Comment by bmilkov** [comment]
Sweet! Looks good

File: monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+## Preparation
+
+The first step is to update all the pis' package repositories, install all dependencies, make all necessary directories, and copy files to the right locations.
+
+### On the VM
+
+1. Make the /usr/bin/piman directory and copy the monitoring directory there:
+
+```
+sudo mkdir /usr/bin/piman
```
Comment (breed) [inline monitoring/README.md]: this is not a good place to put it. perhaps /usr/local/piman/bin or /usr/share/piman/bin

File: monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+```
+sudo pip3 install requests
+```
+
+### On the Pis
+1. Copy the files monitoring-server.py and pi-server-monitoring.service to each pi's home directory:
+
+```
+cd /path/to/piman/monitoring
```
Comment (breed) [inline monitoring/README.md]: this was the directory above, right?

**Review by breed — CHANGES_REQUESTED** [review]
there was a change that went in for monitoring install scripts on the pis. can you reference those instead of walking through the steps.

**Comment by antimony123** [comment]
@breed I updated the docs to reflect the requested changes as well as the automation scripts. I also updated monitoring_piman.sh to use the better directory, /usr/local. Let me know if there is anything else to change. Thanks!

File: monitoring/README.md [hunk 3]
```diff
@@ -37,68 +37,34 @@ The first step is to update all the pis' package repositories, install all depen
...
-
-Repeat this for all other pis.
-
-2. SSH into one of the pis:
-
+1. Run the following script to initialize the pis, copy all needed files to them, and install and run the monitoring service on them:
 ```
-ssh pi@<ip address>
+cd /usr/local/piman/monitoring
+sudo ./init_pis.sh pi piman /usr/local/piman/monitoring/hosts.csv /usr/local/piman/monitoring
```
Comment (breed) [inline monitoring/README.md]: this doesn't seem correct. isn't hosts.csv in /usr/local/piman?

**Review by breed — CHANGES_REQUESTED** [review]

File: monitoring/README.md (hunk 3 above)
Comment (antimony123) [reply monitoring/README.md]: It is indeed. I fixed the path to be /usr/local/piman/hosts.csv.

**Review by breed — APPROVED** [review]
looks good

**Pull request merged**